scrunch/datasets.py

```diff
@@ -1088,6 +1044,47 @@ def editor(self, _):
             'Unsupported operation on the editor property'
         )
 
+    def change_editor(self, user):
+        """
+        Change the current editor of the Crunch dataset.
+
+        :param user:
+            The email address, the crunch url or a User instance of the user
+            who should be set as the new current editor of the given dataset.
+
+        :returns: None
+        """
+        def _host_from_url(url):
+            resolved = urlsplit(url)
+            return resolved.hostname
+
+        def _to_url(email):
+            api_users = 'https://{}/api/users/'.format(
+                _host_from_url(self.resource.self)
+            )
+            user_url = None
+
+            users = self.resource.session.get(api_users).payload['index']
```

Comment:
Now that you have the root resource, if you have the email you can get the user like this:

```
user = root.users.by('email')[email]`
```

Then do `self.resource.edit(current_editor=user.entity_url)`